torchdata/dataloader2/reading_service.py

```diff
@@ -187,14 +137,22 @@ class PrototypeMultiProcessingReadingService(ReadingServiceInterface):
             of each epoch in each worker process with ``WorkerInfo``
             and ``DataPipe`` as the expected arguments.
 
+    Note:
+        - This ``ReadingService`` is still in prototype mode and will replace
+          :class:`MultiProcessingReadingService`.
+        - It currently does both distributed and multiprocessing sharding over the pipeline.
+          The distributed-related code is going to be removed when ``SequentialReadingService``
```

Comment:
any doc about `SequentialReadingService` and `DistributedReadingService`? :)

Reply:
Sequential: https://fburl.com/gdoc/bwmue33j
Distributed: https://pytorch.org/data/beta/generated/torchdata.dataloader2.DistributedReadingService.html#torchdata.dataloader2.DistributedReadingService
And, distributed is quite simple by doing two things:
- Add fullsync
- Launch distributed Process Group to share distributed random seeds